src/state/slices/common-selectors.ts

```diff
@@ -38,23 +38,32 @@ export const selectWalletEnabledAccountIds = createDeepEqualOutputSelector(
   },
 )
 
-export const selectWalletAccountIds = createDeepEqualOutputSelector(
+export const selectEnabledWalletAccountIds = createDeepEqualOutputSelector(
```

Comment:
enabled vs. enabled`Wallet` above, meaning this selects `enabled` AccountIds which are in portfolio.wallet.byId.

In comparison, the above selects `enabledAccountIds[walletId]`, but these may not be present in `portfolio.wallet.byId` (yet)